pulp_2_tests/tests/rpm/api_v2/test_download_policies.py

```diff
@@ -270,18 +274,22 @@ def setUpClass(cls):
         # existing units.
         reset_pulp(cls.cfg)
 
-        # Create, sync and publish a repository.
-        repo = _create_repo(cls.cfg, 'on_demand')
-        cls.resources.add(repo['_href'])
-        sync_repo(cls.cfg, repo)
+        # Create and sync a repository.
+        api_client = api.Client(cls.cfg, api.json_handler)
+        cls.repo = {}
```

Comment:
IS this `cls.repo = {}` created here to ensure it exists on TearDown?  I suggest to change teardown (commented there) instead of creating this unused variable here.

Reply:
Yeah. I will address your comments. Thanks.